### ISSUE TITLE
Possible issue with jqwrapper.js?

### ISSUE THREAD

**Issue by RCatersels** [AuraQ/ClockPickerForMendix#2]
We just tried the model with this widget in the Mendix cloud but it seems that the jqwrapper.js can not be found and hence the start page of the model will not load. Could it be that the Mendix cloud is blocking this Javascript library?

Did you already try your widget in the Mendix cloud?

Regards,

Ronald

**Comment by JelteMX** [comment]
If think it has to do with 

``` js
define(['./jquery-1.11.2'], function(jQuery) {
```

that should be 

``` js
define(['ClockPickerForMendix/lib/jquery-1.11.2'], function(jQuery) {
```

(That's my best quess right now)

**Comment by lindski** [comment]
Been scratching my head over this for a while but have finally got to the bottom of the problem... turned out to be a case sensitivity issue... my paths in the packages array were referencing a widget named 'ClockPickerForMendix' but the correct case for the widget is 'ClockPickerforMendix' (note the lower case f).

I've just committed a fix for this (17cdf94). 

I'll create a new release and submit an update to the app store (the dist folder contains the updated mpk if you want to take a copy sooner)

**Comment by RCatersels** [comment]
I just checked it and new version is working correctly now.
